<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlowWindow</class>
 <widget class="QMainWindow" name="FlowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OVSDB Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="unset">
       <attribute name="title">
        <string>Awaiting update...</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10"/>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="ssgCheckBox">
        <property name="text">
         <string>Server-side grep</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ssgComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxVisibleItems">
         <number>20</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
        <property name="minimumContentsLength">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ssgSaveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ssgDeleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="hostLabel">
        <property name="text">
         <string>Host</string>
        </property>
        <property name="buddy">
         <cstring>hostComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="hostComboBox">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="intervalCheckBox">
        <property name="text">
         <string>Auto-refetch every</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="intervalSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fetchPathsButton">
        <property name="toolTip">
         <string>Refetches the datapath names and rebuilds the window tabs to reflect them.  Use when the network has been reconfigured, e.g. a bond has been created</string>
        </property>
        <property name="text">
         <string>Refetch Datapath List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fetchButton">
        <property name="text">
         <string>Refetch</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_DB_Window"/>
    <addaction name="actionNew_Flow_Window"/>
    <addaction name="actionShow_Log"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShow_Log">
   <property name="text">
    <string>Show Log</string>
   </property>
  </action>
  <action name="actionNew_DB_Window">
   <property name="text">
    <string>New DB Window</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew_Flow_Window">
   <property name="text">
    <string>New Flow Window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
